python_modules/dagster/dagster/_core/definitions/auto_materialize_condition.py

```diff
@@ -45,6 +45,8 @@ class ParentMaterializedAutoMaterializeCondition(NamedTuple):
     """
 
     decision_type: AutoMaterializeDecisionType = AutoMaterializeDecisionType.MATERIALIZE
+    updated_asset_keys: Optional[FrozenSet[AssetKey]] = None
+    will_update_asset_keys: Optional[FrozenSet[AssetKey]] = None
```

Comment:
we might as well track these separately, can always decide not to distinguish in the ui